src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import 'variables/variabrles';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: $font-weight4;
```

Comment:
this is a standard font.
That is, if u don't use any `font-weight` property - the boldness will be 400 according to the default.
remove it and this variable